Documentation/HowToGuides/CreatingAnArtificalButton/README.md

```diff
@@ -0,0 +1,111 @@
+#  Creating An Artifical Button
+
+> * Level: Intermediate
+>
+> * Reading Time: 10 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+
```

Comment:
An artificial button is ideal for a simple non physics enabled button that just has the basic action of hitting something and having it push in and pop out. This can be achieved using the Linear Transform Drive along with a Collider Tracker and some clever use of the UnityEvents available.